src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpSettings.cs

```diff
@@ -295,12 +295,13 @@ private ServiceBusProcessorOptions DetermineMessageProcessorOptions()
             {
                 // Assign the configured defaults
                 messageHandlerOptions.AutoCompleteMessages = Options.AutoComplete;
-                messageHandlerOptions.MaxConcurrentCalls = Options.MaxConcurrentCalls ?? messageHandlerOptions.MaxConcurrentCalls;
+                messageHandlerOptions.MaxConcurrentCalls = Options.MaxConcurrentCalls;
+                messageHandlerOptions.PrefetchCount = Options.PrefetchCount;
```

Comment:
Since this is indeed functionality that is out of our hands, it would be a bit too much to test out Microsoft's functionality in integration tests.

Reply:
Seems like I missed that one.